example_case_folders/08_floris_standin_only/run_script.sh

```diff
@@ -1,49 +1,35 @@
 #!/bin/bash
 
-# Kill any active helics jobs
-source ../find_and_kill_helics
-
-# Determine the base path for Conda initialization
-if [ -f "/home/$USER/anaconda3/etc/profile.d/conda.sh" ]; then
-    # Common path for Anaconda on Linux
-    CONDA_PATH="/home/$USER/anaconda3/etc/profile.d/conda.sh"
-elif [ -f "/Users/$USER/anaconda3/etc/profile.d/conda.sh" ]; then
-    # Common path for Anaconda on macOS
-    CONDA_PATH="/Users/$USER/anaconda3/etc/profile.d/conda.sh"
-elif [ -f "/opt/anaconda3/etc/profile.d/conda.sh" ]; then
-    # Alternative system-wide installation path
-    CONDA_PATH="/opt/anaconda3/etc/profile.d/conda.sh"
-elif [ -f "opt/miniconda3/etc/profile.d/conda.sh" ]; then
-    # Alternative system-wide installation path
-    CONDA_PATH="/opt/miniconda3/etc/profile.d/conda.sh"
-elif command -v conda &> /dev/null; then
-    # If conda is in PATH, use the which command to find conda location
-    CONDA_PATH=$(dirname "$(which conda)")/../etc/profile.d/conda.sh
-else
-    echo "Conda installation not found. Please ensure Conda is installed and in your PATH."
-    exit 1
-fi
-
-# Source the Conda initialization script
-source "$CONDA_PATH"
-conda activate hercules
+# Locate the scripts folder
+SCRIPTS_DIR="../../scripts"
```

Comment:
if you run the script with bash though?  I'm not sure,

Reply:
Agreed, perhaps we should just be encouraging Windows users to use WSL... presumably, users could try to run things in the Command Prompt, and I'm not sure it would work.

Reply:
Sounds good to me